common/src/WooCommerce.php

```diff
@@ -14,6 +16,21 @@ public function __construct(BasePlugin $plugin) {
         $this->plugin = $plugin;
         add_action('woocommerce_order_status_completed', [$this, 'order_completed'], 10, 1);
         add_action('woocommerce_checkout_update_order_meta', [$this, 'set_order_language']);
+        add_action('woocommerce_product_options_sku', [$this, 'addGtinOption']);
+        add_action('woocommerce_admin_process_product_object', [$this, 'saveGtinOption']);
+        register_activation_hook($this->plugin->getPluginFile(), [$this, 'activateSyncReviews']);
```

Comment:
I was not sure when to schedule the `sync_reviews_cron`. Doing it on plugin activation seemed most logical because it will be executed only once. The other option I considered was `add_action('init'..` but this would call it on every page load.

Reply:
The cronjob should get executed when you run wp-cron.php on the cli, no?

Reply:
Yes, it should be executed by `wp-cron.php`. However, this did not work on my local environment, I tried to run from CLI and also with a `GET` request to wp_cron.php.

I tested it on a live website and it worked.

I thought that I did something wrong in setting up the `scheduled_event`. I read that the 'wp_cron.php' is scheduled only when the page is visited so the time could differ depending on the traffic, but this does not seem to be a problem in our case.

Reply:
Yes, I tested it also with `init`, as I mentioned I think the problem has to do with my environment. I looked into the logs and noticed that this shows up when it tries to run the job  `cURL error 7: Failed to connect to localhost port 8080: Connection refused`.
I can see the event registered, also when I run it on a live website it works.

Reply:
Perhaps that's WordPress trying to connect to itself in order to run the cron.  Try setting `DISABLE_WP_CRON` to true in wp-config.php:

~~~
define('DISABLE_WP_CRON', true);
~~~

You are running wp-cron.php on the cli inside vagrant/docker right?